### ISSUE TITLE
Center Points Matching Strategy

### ISSUE THREAD

**Issue by Sissi-lu** [HiLab-git/SCPM-Net#8]
Hi, I am very interested in your work!  
I tried your code on the previous structure DeepLung released on Github. I found the output of the SCPM-Net is C1, R1, off1, C2, R2, and off2. But how do they refuse together? I did not find it out.
And the Center Point matching strategy is essential to select positive and negative samples so that the loss in the training process can be calculated.  So could you release your code of the Top-K center point matching strategy? I did not figure out the matching strategy.
Thanks a lot.
Look forward to hearing from you~!

**Comment by Luoxd1996** [comment]
Hi,
Thanks for your attention, the training code please check this project:https://github.com/zunzhumu/CPMNetv2.
Best,
Xiangde Luo.
